pom.xml

```diff
@@ -1468,8 +1468,8 @@
     <powermockVersion>2.0.9</powermockVersion>
     <okhttpVersion>3.10.0</okhttpVersion>
     <okioVersion>1.14.0</okioVersion>
-    <opennmsApiVersion>0.6.0</opennmsApiVersion>
-    <opennmsApiVersionOsgi>0.6.0</opennmsApiVersionOsgi>
+    <opennmsApiVersion>0.7.0-SNAPSHOT</opennmsApiVersion>
```

Comment:
Updating OIA to 0.7.0-SNAPSHOT - we expect further updates before cutting a release.